src/transformers/models/chameleon/modeling_chameleon.py

```diff
@@ -1449,9 +1689,10 @@ def _update_causal_mask(
 class ChameleonForConditionalGeneration(ChameleonPreTrainedModel):
     _tied_weights_keys = ["lm_head.weight"]
 
-    def __init__(self, config):
+    def __init__(self, config: ChameleonConfig):
         super().__init__(config)
         self.model = ChameleonModel(config)
+        self.vocabulary_mapping = self.model.vocabulary_mapping
```

Comment:
why we need this? I don't see where it's used

Reply:
It's now being used in the `generate` method